enterprise/migrator/migrator.go

```diff
@@ -104,8 +102,8 @@ func (migrator *MigratorT) setupFileManager() filemanager.FileManager {
 		conf["accessKey"] = config.GetEnv("MIGRATOR_SECRET_ACCESS_KEY", "")
 	case "GCS":
 		credentialsFilePath := config.GetRequiredEnv("GOOGLE_APPLICATION_CREDENTIALS")
-		credentials, error := ioutil.ReadFile(credentialsFilePath)
-		if error != nil {
+		credentials, err := ioutil.ReadFile(credentialsFilePath)
```

Comment:
`ioutil.ReadFile` is deprecated, please use `os.ReadFile` instead

Reply:
Sure